# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/src/InputBlockState.h
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 gfx/layers/apz/src/InputBlockState.h
--- a/gfx/layers/apz/src/InputBlockState.h	Sun May 26 14:29:56 2019 +0000
+++ b/gfx/layers/apz/src/InputBlockState.h	Sun May 26 14:30:14 2019 +0000
@@ -107,9 +107,9 @@
 
   // The APZC that was actually scrolled by events in this input block.
   // This is used in configurations where a single input block is only
-  // allowed to scroll a single APZC (configurations where gfxPrefs::
-  // APZAllowImmediateHandoff() is false).
-  // Set the first time an input event in this block scrolls an APZC.
+  // allowed to scroll a single APZC (configurations where
+  // StaticPrefs::APZAllowImmediateHandoff() is false). Set the first time an
+  // input event in this block scrolls an APZC.
   RefPtr<AsyncPanZoomController> mScrolledApzc;
 
  protected: